packer/answer_files/2008_r2/Autounattend.xml

```diff
@@ -272,6 +272,12 @@
                     <Order>99</Order>
                     <RequiresUserInput>true</RequiresUserInput>
                 </SynchronousCommand>
+                <SynchronousCommand wcm:action="add">
+                    <CommandLine>cmd.exe /c "set PACKER_BUILDER_TYPE=hyperv-first-boot&&a:\vm-guest-tools.bat</CommandLine>
```

Comment:
This is an xml file, the diff here is pointing out that `&` is a special character in most XML parsers. Should these be `&amp;`?

Reply:
Nope, raw & is the way to go. I tried it as &amp; and it doesn't produce the expected result.